lib/Notification/Notifier.php

```diff
@@ -73,6 +73,18 @@ public function prepare(INotification $notification, $languageCode) {
 		$params = $notification->getSubjectParameters();
 
 		switch ($notification->getSubject()) {
+			case 'card-assigned':
+				$cardId = $notification->getObjectId();
+				$boardId = $this->cardMapper->findBoardId($cardId);
+				$initiator = $this->userManager->get($params[2]);
+				if ($initiator !== null) {
+					$params[2] = $initiator->getDisplayName();
+				}
+				$notification->setParsedSubject(
+					(string) $l->t('The card "%s" on "%s" has been assigned to you by %s.', $params)
+				);
```

Comment:
Can you add a rich subject as well, similar to https://github.com/nextcloud/deck/pull/545/files#diff-52c5c003756b8dcc9c5318257107346dL95? This way the user will also be linked properly

Reply:
Done